src/ProductAttributePageTitle.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Allows optional customization of page title.
+ */
+class ProductAttributePageTitle {
+  public static function init() {
+    // Register ACF field.
+    ProductAttributePageTitle::register_acf_page_title($group_filter);
+
+    // Front-end output of title.
+    add_filter('woocommerce_page_title', __CLASS__.'::woocommerce_page_title');
+  }
+
+  /**
+   * Register ACF field.
+   */
+  public static function register_acf_page_title($group_filter) {
+    register_field_group([
+      'key' => 'acf_group_page_title',
+      'title' => __('Listing Page', Plugin::L10N),
+      'fields' => [
+        [
+          'key' => 'acf_field_page_title',
+          'label' => __('Page Title', Plugin::L10N),
+          'name' => 'page_title',
+          'type' => 'text',
+          'instructions' => __('Only changes how the name appears on the page. Leaves the filter drop-down menu alone.', Plugin::L10N),
+          'required' => 0,
+          'conditional_logic' => 0,
+          'wrapper' => [
+            'width' => '',
+            'class' => '',
+            'id' => '',
+          ],
+          'default_value' => '',
+          'placeholder' => '',
+          'prepend' => '',
+          'append' => '',
+          'maxlength' => '',
```

Comment:
I think there is lots of stuff which can be dropped as it's the default. E.g. all of this code + probably more. 
```suggestion
```